twopane-navigation/src/components/paneRenderer/PaneRenderer.tsx

```diff
@@ -73,12 +73,23 @@ const PaneRenderer = (props: IPaneRendererProps) => {
         
     }
 
-// TODO MOVE TO ONE RENDER
+    const isAccessible = (component: IPaneComponent) => {
+        const isLastPaneOne = component.pane === paneType.ONE && paneComponents.filter(x => x.pane === paneType.ONE).pop()?.key === component.key;
+        const isLastPaneTwo = component.pane === paneType.TWO && paneComponents.filter(x => x.pane === paneType.TWO).pop()?.key === component.key;
+
+        if ((paneRects.length > 1 && (isLastPaneOne || isLastPaneTwo && !isPaneOneExtended())) || (paneRects.length === 1 && isLastPaneOne)) {
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+
     return (
         <Fragment>
             {
                 paneComponents.map((val: IPaneComponent) =>
-                <View key={val.key}>
+                    <View importantForAccessibility={isAccessible(val) ? 'auto' : 'no-hide-descendants'} key={val.key}>
```

Comment:
nit: whitespace.

Reply:
What whitespace are you referring to?

Reply:
git is highlighting the whitespace before View tags on line 92 and 120, I'm guessing the indentation is one too "deep"